.github/workflows/build-master.yml

```diff
@@ -48,4 +48,4 @@ jobs:
         pip install .[test]
     - name: Lint with flake8
       run: |
-        flake8 aicsimageio --count --verbose --max-line-length=127 --show-source --statistics
```

Comment:
I would try to leave these out of this PR, just totally unrelated changes

Reply:
Reverted the commit